README.md

```diff
@@ -44,3 +44,11 @@ From within the root directory:
 npm install
 ```
 
+## API
+
+|  Method      |  Endpoint                           |  Action                                                |
+| ------------ | ----------------------------------- | ------------------------------------------------------ |
+|  **POST**    |  /api/related_products/*PID*/       |  Creates record *PID*                                  |
+|  **GET**     |  /api/related_products/*PID*        |  Response body contains related products for *PID*     |
+|  **PATCH**   |  /api/related_products/*PID*/*SID*  |  Adds a recommendation for *SID* to record *PID*       |
+|  **DELETE**  |  /api/related_products/*PID*/*SID*  |  Removes a recommendation for *SID* from record *PID*  |
```

Comment:
What is PID & SID?

Reply:
One for the primary record, one for the id of the recommended item (one of the other 9,999,999 records)